docs/api.rst

```diff
@@ -22,3 +27,13 @@ series
 ------
 
 .. autoclass:: series
+
+signal
+------
+
+.. autoclass:: signal
+
+spectrum
+--------
+
+.. autoclass:: spectrum
```

Comment:
The same as for `signal()`, please change to `autofunction`

Reply:
done